macros/rust

```diff
@@ -11,6 +11,9 @@
 # Enable optimization, debuginfo, and link hardening.
 %__global_rustflags -Copt-level=3 -Cdebuginfo=2 -Ccodegen-units=1 -Cforce-frame-pointers=yes -Clink-arg=-Wl,-z,relro,-z,now
 
+# Disable optimization but keep debuginfo and link hardening
+%__global_rustflags_slow -Copt-level=3 -Cdebuginfo=2 -Cforce-frame-pointers=yes -Clink-arg=-Wl,-z,relro,-z,now
```

Comment:
The comment is wrong; we're not disabling optimization entirely, just one specific aspect of that.

I don't love "slow" as a suffix - it's actually slower to build with fewer codegen units, and in practice where we use these flags it's because we want the crate to build faster. This is the opposite of what the macro name implies.

Let's just use "aws_sdk" as the suffix. It's annoying to encode crate specifics into a macro but we end up doing that for "--features fips" so I can live with it to cut down on boilerplate elsewhere.

I would prefer to structure this as:
```
%__global_rustflags_base -Copt-level=3 -Cdebuginfo=2 -Cforce-frame-pointers=yes -Clink-arg=-Wl,-z,relro,-z,now
%__global_rustflags -Ccodegen-units=1 %__global_rustflags_base
%__global_rustflags_aws_sdk %__global_rustflags_base
```